applications/luci-app-eoip/Makefile

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Bogdan Shatik (bogdikxxx@mail.ru)
+# This is free software, licensed under the Apache License, Version 2.0
+
+include $(TOPDIR)/rules.mk
+
+PKG_MAINTAINER:=Bogdan Shatik <bogdikxxx@mail.ru>
+PKG_NAME:=luci-app-eoip
+PKG_VERSION:=1.0.1
+
```

Comment:
You should add the license database line. Your copyright sets is as Apache 2

`PKG_LICENSE:=Apache-2.0`